db/users.js

```diff
@@ -7,10 +7,10 @@ async function createUser ({ firstname, lastname, email, imgURL, username, passw
     try {
         const { rows: [user] } = await client.query(`
         INSERT INTO users(firstname, lastname, email, "imgURL", username, password, "isAdmin", address) 
-        VALUES ($1, $2)
+        VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
         ON CONFLICT (username) DO NOTHING
-        RETURNING id, username
-        `, [username, hashedPassword]);
+        RETURNING *
```

Comment:
we should either return explicit fields (`RETURNING id, username, ...`) or make sure we delete the password before we return.